viewing_party/party.py

```diff
@@ -1,23 +1,225 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from ast import If
+from types import NoneType
+from tests.test_constants import MOVIE_TITLE_1, USER_DATA_2
+
+
+def create_movie(movie_title, genre, rating):
+
+
+    movie = {}
+
+    movie["title"] = movie_title
+    movie["genre"] = genre
+    movie["rating"] = rating
+    
+    for item in movie.values():
+        if item == None:
+            return None
+    else:
+        return movie
+
+def add_to_watched(user_data, movie):
+
+    user_data = {
+    "watched": []
+    }   
+    copy = movie.copy()
+
+    user_data["watched"].append(copy)
+    
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data = {
+        "watchlist": []
+    }
+    copy1 = movie.copy()
+    
+    user_data["watchlist"].append(copy1)
+
+    return user_data
+
+def watch_movie(user_data, title):
+
+
+
+    for movie in user_data["watchlist"]:
+        if title == movie['title']:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)
+    return user_data
+            
+
+        
+        
+
+
+
+

```

Comment:
Brooke, mind your space!  Please take up space everywhere in life, except in your functions LOL. One line of blank space is enough.